db/gxasc-schema.sql

```diff
@@ -42,6 +42,33 @@ CREATE TABLE scxa_cell_clusters
   PRIMARY KEY (experiment_accession, k, cell_id)
 );
 
+CREATE TABLE scxa_cell_group
```

Comment:
All these changes should go on a new file on the migrations part, not in the baseline schema.

Reply:
Yep, I was more using this to illustrate the structure I wanted to go for, for discussion, don't actually want to merge this.

Reply:
I'm overall fine with the structure, but we need to see how it constraints both the loading and querying (and whether for instance the required JOIN on query time is worth the space saving due to normalisation). We could make a ball park estimate on the size of the denormalised table vs normalised for the largest experiments.

Reply:
@alfonsomunozpomer - do you have any feelings on this? I was always taught to normalise in this sort of situation, but I have no feeling for the performance implications here.

Reply:
I would also suggest that we open a parallel PR on db-scxa where you start sketching how the loading will work, including testing some simple examples, which in turn will provide things to keep in mind for the structure being proposed here.

Reply:
our primary objective should be best perfomance at query time, followed by best perfomance at loading time, followed by size considerations and then db structure correctness (IMHO). When any of this is massively bad, it should take slight precedence over the others until fixed I would say. And query time should never be bad at all. Currently loading time for clustering data is slow in the current setup for large experiments.

Reply:
> I would also suggest that we open a parallel PR on db-scxa where you start sketching how the loading will work, including testing some simple examples, which in turn will provide things to keep in mind for the structure being proposed here.

This ongoing at https://github.com/ebi-gene-expression-group/db-scxa/pull/40. I've made proposals for loading of clusterings, markers next.